CONTRIBUTING.rst

```diff
@@ -109,14 +108,14 @@ a new pull request against the `corresponding docs section of the MLflow code ba
 
 For more information about Plugins, see https://mlflow.org/docs/latest/plugins.html.
 
-Developing and testing changes to MLflow
-########################################
+Developing and testing changes for MLflow
+#########################################
 The majority of the MLflow codebase is developed in Python. This includes the CLI, Tracking Server,
 Artifact Repositories (e.g., S3 or Azure Blob Storage backends), and of course the Python fluent,
 tracking, and model APIs.
 
-Prerequisites
-~~~~~~~~~~~~~
+General prerequisites and dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 First, ensure that your name and email are
 `configured in git <https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup>`_ so that
 you can `sign your work`_ when committing code changes and opening pull requests:
```

Comment:
Do you think it would be good to say why we ask them to ask their work?

Reply:
We mention that in the[ Sign your work](https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.rst#sign-your-work) section: " You need to sign that you wrote the patch or otherwise have the right to pass it on as an open-source patch."